src/components/MintDataCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { ArrowForwardIcon, LinkIcon } from '@chakra-ui/icons';
+import { Box, HStack, Link, VStack } from '@chakra-ui/react';
+
+interface MintData {
+  address: string;
```

Comment:
こちらも同様に`address`→`to`だと思います！

```suggestion
  to: string;
```